Task_1/Manage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public class Manage
+    {
+        public void Run()
+        {
+            Random rnd = new Random();
+
+            Pc newPc = new Pc(Convert.ToBoolean(rnd.Next(2)));
```

Comment:
It was important to use interface references, not Pc and Monitor but ISwitchable and invoke methods from interface reference.